pkg/apis/pipeline/v1/pipeline_validation.go

```diff
@@ -212,6 +212,17 @@ func (pt PipelineTask) Validate(ctx context.Context) (errs *apis.FieldError) {
 		NamespacedTaskKind: true,
 		ClusterTaskRefKind: true,
 	}
+
+	if pt.OnError != "" {
+		errs = errs.Also(version.ValidateEnabledAPIFields(ctx, "OnError", config.AlphaAPIFields))
+		if pt.OnError != PipelineTaskContinue && pt.OnError != PipelineTaskStopAndFail {
+			errs = errs.Also(apis.ErrInvalidValue(pt.OnError, "OnError", "PipelineTask OnError must be either \"continue\" or \"stopAndFail\""))
+		}
+		if pt.OnError == PipelineTaskContinue && pt.Retries > 0 {
+			errs = errs.Also(apis.ErrMultipleOneOf("OnError", "Retries"))
```

Comment:
Would this be a bit confusing cuz we don't allow PipelineTaskContinue and Retries, but the error is `ErrMultipleOneOf("OnError", "Retries"))`, which makes users think they cannot set OnError and Retries both

Reply:
Thanks! I have made the error msg more explicit